src/HttpHdrCc.cc

```diff
@@ -60,7 +60,7 @@ void
 httpHdrCcInitModule(void)
 {
     // check invariant on initialization table
-    for (unsigned int j = 0; CcAttrs[j].name != nullptr; ++j) {
+    for (int j = 0; CcAttrs[j].name != nullptr; ++j) {
         assert (static_cast<int>(CcAttrs[j].id) == j);
```

Comment:
This change replaced the wrong type AFAICT: `CcAttrs[j].id` is unsigned because HttpHdrCcType is unsigned. Thus, the loop variable type was OK, but the static_cast was wrong.

```suggestion
    for (unsigned int j = 0; CcAttrs[j].name != nullptr; ++j) {
        assert (static_cast<unsigned int>(CcAttrs[j].id) == j);
```

Reply:
ok